contrib/opencensus-ext-azure/opencensus/ext/azure/trace_exporter/__init__.py

```diff
@@ -75,6 +76,21 @@ def span_data_to_envelope(self, sd):
                 sd.parent_span_id,
             )
         if sd.span_kind == SpanKind.SERVER:
+            if 'error.message' in sd.attributes:
```

Comment:
Can we use `attributes_helper.COMMON_ATTRIBUTES['ERROR_NAME']` instead of a hardcoded string?